content/webapp/components/IIIFViewer/MainViewer.tsx

```diff
@@ -292,40 +321,57 @@ const ItemRenderer = memo(({ style, index, data }: ItemRendererProps) => {
         </MessageContainer>
       ) : (
         <>
-          {!mainLoaded && <LL $lighten={true} />}
-          {(imageType === 'main' || mainLoaded) &&
-            urlTemplateMain &&
-            infoUrl && (
-              <>
-                {overlayPositionData &&
-                  overlayPositionData.map((item, i) => {
+          {overlayPositionData &&
+            overlayPositionData.map((item, i) => {
+              return (
+                <SearchTermHighlight
+                  key={i}
+                  $top={item.overlayTop}
+                  $left={item.overlayLeft}
+                  $width={item.highlight.w}
+                  $height={item.highlight.h}
+                  $rotation={item.rotation}
+                />
+              );
+            })}
+
+          {displayItems.map((item, i) => {
```

Comment:
Is it worth displaying anything if displayItems is empty? (I think at this point it's me not being 100% on what everything is because of my lack of knowledge on this area of this website!)

Reply:
possibly? I'm not sure it would ever really happen and this doesn't change current behaviour so I'll leave as is for now.